conda-env/dev.yml

```diff
@@ -0,0 +1,31 @@
+# Conda pcmdi_metrics development environment
+name: pcmdi_metrics_dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  # Base
+  # ==================
+  - python=3.8.10
+  - pip=21.2.4
+  - numpy=1.21.3
+  - matplotlib=3.4.3
+  - cdat::vcs=8.2.1
+  - cdat::vcsaddons=8.2.1
```

Comment:
Are these `vcs` dependencies still required? I noticed they were imported in some files, but #774 doesn't list `vcs`.